x-pack/plugins/synthetics/server/routes/pings/journeys.ts

```diff
@@ -6,9 +6,13 @@
  */
 
 import { schema } from '@kbn/config-schema';
-import { API_URLS, SYNTHETICS_API_URLS } from '../../../common/constants';
+import { getJourneyFailedSteps } from '../../legacy_uptime/lib/requests/get_journey_failed_steps';
+import { SYNTHETICS_API_URLS } from '../../../common/constants';
 import { UMServerLibs } from '../../legacy_uptime/uptime_server';
-import { UMRestApiRouteFactory } from '../../legacy_uptime/routes/types';
+import {
+  SyntheticsRestApiRouteFactory,
+  UMRestApiRouteFactory,
+} from '../../legacy_uptime/routes/types';
```

Comment:
Might be good to get SyntheticsRestApiRouteFactory type out of legacy_uptime at some point.